K2Bridge/CodeAnalysis.ruleset

```diff
@@ -98,6 +98,7 @@
     <Rule Id="SA1202" Action="Error" />
     <Rule Id="SA1203" Action="Error" />
     <Rule Id="SA1204" Action="Error" />
+    <Rule Id="SA1210" Action="None" />
```

Comment:
this is about using statements order.
why not sort?

Reply:
The SA1210 is in conflict with the System.Linq ordering rule